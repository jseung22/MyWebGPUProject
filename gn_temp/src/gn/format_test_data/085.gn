# Suppress deduplication based on comment.

# KEEPDUPS
sources = []

# KEEPDUPS
sources = [ "a" ]

# KEEPDUPS
sources += [ "a" ]

# KEEPDUPS
# This is KEEPDUPS because reason.
sources = [
  "z",
  "a",
  "z",
  "z2",
  "a",
  "y.cc",
]

# This is KEEPDUPS because reason:
# KEEPDUPS
sources += [
  "z",
  "a",
  "z",
  "z2",
  "a",
  "y.cc",
]
