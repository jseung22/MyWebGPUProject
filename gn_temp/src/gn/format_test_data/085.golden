# Suppress deduplication based on comment.

# KEEPDUPS
sources = []

# KEEPDUPS
sources = [ "a" ]

# KEEPDUPS
sources += [ "a" ]

# KEEPDUPS
# This is KEEPDUPS because reason.
sources = [
  "a",
  "a",
  "y.cc",
  "z",
  "z",
  "z2",
]

# This is KEEPDUPS because reason:
# KEEPDUPS
sources += [
  "a",
  "a",
  "y.cc",
  "z",
  "z",
  "z2",
]
